src/addons/messagelog/messagelog-db/src/main/java/ee/ria/xroad/messagelog/database/MessageRecordEncryption.java

```diff
@@ -56,10 +56,9 @@
 import java.util.Map;
 
 @Slf4j
-public final class MessageLogEncryption {
+public final class MessageRecordEncryption {
```

Comment:
Would be good to create a basic Javadoc explaining what this class does. I'm not sure why Checkstyle is not complaining about it.